build.py

```diff
@@ -86,5 +86,7 @@
         'vendor/tree-sitter-typescript/typescript',
         'vendor/tree-sitter-verilog',
         'vendor/tree-sitter-yaml',
+        'vendor/tree-sitter-hcl',
```

Comment:
I was trying to get inspiration from PR #11, but it doesn't have an entry in this file!
What's up with the RST language of PR #11) not having an entry here? 

I see there's #15, could that be related?